config/izum_vega.py

```diff
@@ -1,16 +1,5 @@
-from os import environ, makedirs
-
-from eessi.testsuite.constants import *
-
-# Get username of current user
-homedir = environ.get('HOME')
-
-# This config will write all staging, output and logging to subdirs under this prefix
-reframe_prefix = f'{homedir}/reframe_runs'
-log_prefix = f'{reframe_prefix}/logs'
-
-# ReFrame complains if the directory for the file logger doesn't exist yet
-makedirs(f'{log_prefix}', exist_ok=True)
```

Comment:
this was not needed for me, and ReFrame did not complain.
maybe you used an older version?

Reply:
Not sure, could be. Are you sure the prefix didn't exist already in your case though when you were testing? I mean, ReFrame for sure creates the directory structure _within_ the prefix. The issue (at least previously) was only if the prefix itself didn't exist yet.

I'm also fine taking it out. If the problem shows it's head again, it should be clear enough anyway.

Reply:
i re-tested this with a non-existing prefix to be sure, and it worked fine.

Reply:
Ok, it can go then :)